docusaurus.config.js

```diff
@@ -79,6 +79,21 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      image: "https://botkube.io/images/botkube-title-social.png",
```

Comment:
As stated here: https://docusaurus.io/docs/api/themes/configuration#meta-image, we can use the relative path:

```suggestion
      image: "images/botkube-title-social.png",
```

I'd avoid hardcoding the domain in too many places, especially if we know that the docs site can be moved under subdomain in near future 🙂

Reply:
Yeah I wasn't sure if we can use relative path. But yeah thanks, fixed.